tools/system_libs.py

```diff
@@ -1203,7 +1203,7 @@ def get_files(self):
       'stdlib_stdexcept.cpp',
       'stdlib_typeinfo.cpp',
       'private_typeinfo.cpp',
-      'format_exception.cpp',
+      'cxa_emscripten.cpp',
```

Comment:
How about `cxa_exception_emscripten.cpp`?

Also I noticed that the error message was change to `terminating with uncaught exception..` where as before it could be used to get a description of an exception in general.

It seems like it might be useful to be able to decribe and exception without it being the an uncaught one.   So we might need two different functions here.. one can call the other?

Reply:
I thought about this, but is there a way to access Wasm exceptions from JS side unless they are uncaught exceptions? Hmm, but yeah, you're right, someone might want to catch them from JS and not actually terminate the program.. Will fix that in another PR.

Renamed the file to `cxa_exception_emscripten.cpp`.